scan/src/components/data-source/DataSourceExecute.re

```diff
@@ -123,15 +129,15 @@ let resultRender = result => {
           <div className={Styles.resultWrapper(`px(120), `px(12), `auto)}>
             <Text value="Exit Status" color=Colors.gray6 weight=Text.Bold />
           </div>
-          <Text value="0" />
+          <Text value={returncode |> string_of_int} />
         </div>
         <VSpacing size=Spacing.md />
         <div className=Styles.hFlex>
           <HSpacing size=Spacing.lg />
           <div className={Styles.resultWrapper(`px(120), `px(12), `auto)}>
             <Text value="Output" color=Colors.gray6 weight=Text.Semibold />
           </div>
-          <Text value=output code=true weight=Text.Semibold />
+          <Text value={returncode == 0 ? stdout : stderr} code=true weight=Text.Semibold />
```

Comment:
We should show both `stdout` and `stderr`

Reply:
So should we fix the design as well. Because currently we only have "Exit Status" and "Output". Or just combine both of them like stdout ++ stderr.

Reply:
Yes. Please add "Error" field below output.